anatomy/myApp/config/sockets.js.md

```diff
@@ -25,236 +26,144 @@ It provides transparent access to Sails' encapsulated pubsub/socket server for c
-  'browser client expires': 315360000,
-
-  // Does Socket.IO need to GZIP the static files?
-  // This process is only done once and the computed output is stored in memory.
-  // So we don't have to spawn a gzip process for each request.
-  'browser client gzip': false,
-
-  // Optional override function to serve all static files,
-  // including socket.io.js et al.
-  // Of the form :: function (req, res) { /* serve files */ }
-  'browser client handler': false,
-
-  // Meant to be used when running socket.io behind a proxy.
-  // Should be set to true when you want the location handshake to match the protocol of the origin.
-  // This fixes issues with terminating the SSL in front of Node
-  // and forcing location to think it's wss instead of ws.
-  'match origin protocol': false,
-
-  // Direct access to the socket.io MQ store config
-  // The 'adapter' property is the preferred method
-  // (`undefined` indicates that Sails should defer to the 'adapter' config)
-  store: undefined,
-
-  // A logger instance that is used to output log information.
-  // (`undefined` indicates deferment to the main Sails log config)
-  logger: undefined,
-
-  // The amount of detail that the server should output to the logger.
-  // (`undefined` indicates deferment to the main Sails log config)
-  'log level': undefined,
-
-  // Whether to color the log type when output to the logger.
-  // (`undefined` indicates deferment to the main Sails log config)
-  'log colors': undefined,
-
-  // A Static instance that is used to serve the socket.io client and its dependencies.
-  // (`undefined` indicates use default)
-  'static': undefined,
-
-  // The entry point where Socket.IO starts looking for incoming connections.
-  // This should be the same between the client and the server.
-  resource: '/socket.io'
+  //   // (`false` would reject the connection)
+  // },
+
+  /***************************************************************************
+  * Deprecation notice: onConnect() has been removed as of v0.11.0           *
+  * To achieve it's function run a request from the client when socket       *
+  * connects instead                                                         *
+  ***************************************************************************/
+  
+  /***************************************************************************
+  *                                                                          *
+  * `afterDisconnect`                                                        *
+  *                                                                          *
+  * This custom afterDisconnect function will be run each time a socket      *
+  * disconnects                                                              *
+  *                                                                          *
+  * Deprecation notice: This is a replacement for old onDisconnect() function.
```

Comment:
End of line needs a `*` not a `.`, consider breaking the line after `onDisconnect()`